quiz/backend/players/serializers.py

```diff
@@ -10,7 +11,33 @@ class PlayerSerializer(ModelSerializer):
     :rtype: rest_framework.serializers.Serializer
     """
 
+    username = CharField(source="nick", required=True, validators=[validate_profanity])
+
+    class Meta:
+        model = models.Player
+        fields = (
+            "uuid",
+            "username",
+        )
+        read_only_fields = ("uuid",)
+
+
+class PlayerDetailSerializer(ModelSerializer):
+    """Serializer for Player model.
+
+    :returns: PlayerSerializer
+    :rtype: rest_framework.serializers.Serializer
+    """
```

Comment:
I really think that we should work on some Documentation Style Guidelines. Not that there is something wrong here, but I believe that a class isn't returning anything. 😅